app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,7 @@
   </head>
 
   <body>
+    <% if notice %><aside id="notice"><%= notice %></aside><% end %>
```

Comment:
**suggestion:** changing `aside` to `div`

Not sure if that's the proper call, since there's often space for discussion within HTML semantics, but I see aside as something related to the main content of the page, which is often not the case with `notice` (it could be, for instance, a "login" notification".

Source:
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside